app/src/main/java/de/danoeh/antennapod/fragment/SearchFragment.java

```diff
@@ -146,7 +156,16 @@ public void onCreateContextMenu(ContextMenu menu, View v, ContextMenu.ContextMen
         LinearLayoutManager layoutManagerFeeds = new LinearLayoutManager(getActivity());
         layoutManagerFeeds.setOrientation(RecyclerView.HORIZONTAL);
         recyclerViewFeeds.setLayoutManager(layoutManagerFeeds);
-        adapterFeeds = new HorizontalFeedListAdapter((MainActivity) getActivity());
+
+        adapterFeeds = new HorizontalFeedListAdapter((MainActivity) getActivity()) {
+
+            @Override
+            public void onCreateContextMenu(ContextMenu contextMenu, View view,
+                                            ContextMenu.ContextMenuInfo contextMenuInfo) {
+                super.onCreateContextMenu(contextMenu, view, contextMenuInfo);
```

Comment:
Please add `MenuItemUtils.setOnClickListeners(menu, SearchFragment.this::onContextItemSelected);` here. Otherwise the click event might end up in the wrong fragment and therefore change the wrong feed.